mobile/examples/super_resolution/ios/ORTSuperResolution/Podfile

```
@@ -7,8 +7,8 @@ target 'ORTSuperResolution' do
   # Pods for OrtSuperResolution
   pod 'onnxruntime-c'
   
-  # Pre-release version pods
-  pod 'onnxruntime-extensions-c', '0.5.0-dev+261962.e3663fb'
+  # Release version pod for ort-ext
+  pod 'onnxruntime-extensions-c', '0.6.0'
```

just use the latest?
```suggestion
  pod 'onnxruntime-extensions-c'
```